qiskit/opflow/gradients/circuit_qfis/lin_comb_full.py

```diff
@@ -29,35 +30,45 @@
 
 class LinCombFull(CircuitQFI):
     r"""Compute the full Quantum Fisher Information (QFI).
-
     Given a pure, parameterized quantum state this class uses the linear combination of unitaries
     approach, requiring one additional working qubit.
     See also :class:`~qiskit.opflow.QFI`.
     """
 
+    # pylint: disable=signature-differs
     def convert(
         self,
         operator: CircuitStateFn,
         params: Union[ParameterExpression, ParameterVector, List[ParameterExpression]],
+        aux_meas_op: OperatorBase = Z,
+        phase_fix: bool = True,
     ) -> ListOp:
         r"""
         Args:
             operator: The operator corresponding to the quantum state :math:`|\psi(\omega)\rangle`
                 for which we compute the QFI.
             params: The parameters :math:`\omega` with respect to which we are computing the QFI.
+            aux_meas_op: The operator that the auxiliary qubit is measured with respect to.
+                         for aux_meas_op = Z we compute Re[(dω⟨<ψ(ω)|)(dω|ψ(ω)〉)]
+                         for aux_meas_op = -iY we compute Im[(dω⟨<ψ(ω)|)(dω|ψ(ω)〉)]
+            phase_fix: Whether or not to compute the additional phase fix term
+                       Re[(dω⟨<ψ(ω)|)|ψ(ω)><ψ(ω)|(dω|ψ(ω))>]
```

Comment:
I think the indentation must always be 4 spaces for sphinx to pass 😄 

Also one question: the releasenote states the aux op must be `-Y` for the imaginary part but this says `-iY`, which is the right one? 
```suggestion
            aux_meas_op: The operator that the auxiliary qubit is measured with respect to.
                For ``aux_meas_op = Z`` we compute Re[(dω⟨<ψ(ω)|)(dω|ψ(ω)〉)] and
                for ``aux_meas_op = -iY`` we compute Im[(dω⟨<ψ(ω)|)(dω|ψ(ω)〉)].
            phase_fix: Whether or not to compute the additional phase fix term
                Re[(dω⟨<ψ(ω)|)|ψ(ω)><ψ(ω)|(dω|ψ(ω))>].
```